from app import printError as error


def printError(content):
    error(content)
